common/app/routes/Challenges/views/classic/Editor.jsx

```diff
@@ -136,8 +141,9 @@ export class Editor extends PureComponent {
         >
         <NoSSR onSSR={ <CodeMirrorSkeleton content={ content } /> }>
           <Codemirror
-            onChange={ change => classicEditorUpdated(fileKey, change) }
-            options={ this.createOptions({ executeChallenge, mode, cmTheme }) }
+            key={id}
+            onChange={change => classicEditorUpdated(fileKey, change)}
```

Comment:
Sorry for the nit, for the coding style of this code base, we space our props like so
```jsx
<Component
  prop1={ foo }
  prop2={ 'bar' }
  prop3={ 42 }
  prop4={ () => console.log('clicky') }
/>
```

Reply:
sure i will update the PR.